spatialos-sdk/examples/project-example/main.rs

```diff
@@ -58,19 +59,19 @@ fn logic_loop(c: &mut WorkerConnection) {
                         == generated_code::example::Example::component_id()
                     {
                         let component_data =
-                            component::get_component_data::<generated_code::example::Example>(
+                            unsafe { component::get_component_data::<generated_code::example::Example>(
```

Comment:
Can we move these unsafe blocks into the implementation of `get_component_data`?

Reply:
Good point. I'm working on deleting this functions and replacing them with an implementation of `TryFrom` instead, so will probably add the component ID verification to ensure it is safe.